tools/ci/build-test-macos-with-sanitizers.sh

```diff
@@ -31,7 +31,7 @@ $(clang --version)"
 
 msg "building and installing dependencies"
 # tinfo library which is required by iceoryx_introspection isn't available in mac
-brew install ncurses
+brew install ncurses wget
```

Comment:
wget is already installed on the GitHub Runners but you may want to use the script locally.